README.md

```diff
@@ -1,70 +1,113 @@
+- **Rails back-end**: <a name="backend-instructions"></a> This app needs to be connected to an API managing the back-end in order to work properly. So, see the instructions [here](https://github.com/Movgto/hello-rails-back-end) for setting up the back-end application.
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+- **Node.js**: This is mandatory to run the front-end side of the application made with React since we needed for installing the dependencies for the project.
 
-### Code Splitting
+With these prerequisites and configurations in place, you'll be ready to set up and run the app.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+### 📖 Usage <a name="usage"></a>
+Follow these steps to use the app:
 
-### Analyzing the Bundle Size
+In your terminal and the root of the project.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+Run this for installing the dependencies:
 
-### Making a Progressive Web App
+```bash
+npm install
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+And this to start the React application on the local server:
 
-### Advanced Configuration
+```bash
+npm start
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+**Congratulations, now you have both applications running! (Assuming that you also followed these [instructions](#backend-instructions) to run the back-end)**
 
-### Deployment
+Now just open your browser and go to [this link](localhost:3000) and see them in action.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+## 👥 Authors <a name="authors"></a>
 
-### `npm run build` fails to minify
+👤 **Gumaro Monroy Vazquez**
+- GitHub: [GitHub](https://github.com/Movgto)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/gumaro-monroy-vazquez-1705aa165/)
+- WellFound: [Wellfound](https://wellfound.com/u/maro-monroy)
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/Movgto/react-rails-greetings/issues) and contribute to the project.
```

Comment:
- [ ]  Kindly ensure that the issue link is working, I got 404 error in the process of accessing it. 🙏